python/pyspark/sql/connect/functions.py

```diff
@@ -90,7 +90,10 @@ def col(col: str) -> Column:
 
 
 def lit(col: Any) -> Column:
-    return Column(LiteralExpression(col))
+    if isinstance(col, Column):
+        return col
```

Comment:
hmmm I think we should throw exception for this case than let it pass through? 

Basically lit is to create a Column inside which is a literal. The allowed Column here has other possibilities: column reference, sort order, etc. 

Then for example a user might pass a Column(SortOrder) to `lit` and it passes then the user assumes it becomes a Column(literal) which cause problems